card.html

```diff
@@ -4,8 +4,324 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="card.css" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    
+    <div class="card">
+      <div class="card-container">
+        <div class="card-design card-container-front card-design--one">
```

Comment:
You didn't follow the BEM naming.
Instead of this - **card-design--one**
It should have been this - **card-design__one** (the underscore will make one know that **one** is a child of **card design**

There are other classes that are named wrongly like this, so please do the necessary correction

Reply:
Okay I'll make changes to it

Reply:
@VictoriaAde i've corrected it kindly review. Thanks.